app/_posts/blog/2016-05-29-2016-retrospective-planning-for-2017.md

```diff
@@ -0,0 +1,76 @@
+category : blog
+tags : [retrospective, devit2016, devit2017]
+author: Thanasis Polychronakis
+---
+
+
+## DEVit 2016 is over!
+
+Oh what a blast DEVit 2016 was! No matter how exhausted we were we could not stop the rush of endorphins that flooded our bodies for days after. You see, we, and by 'we' I'm referring to everyone that was at DEVit, managed to make a difference, we participated in making history for our community, for our country. We have successfully positioned Greece in the global landscape of international conferences, helped make DEVit a sustainable yearly conference and had such an amazing time!
+
+We know we messed a few things up, I'll get to them one by one in a bit, but what prevailed in the end was our amazing attitude, passion and determination to have a good time. Your amazing vibes gave us all the motivation we needed to continue on with DEVit, scale it and present you with an even more amazing experience.
+
+## Your feedback, the bad staff
+
+As I said before we did mess a few things up... At registration we definitely dropped the ball, at our retrospective meeting we dived deep into all the various reasons why this happened, they were a lot and we had to deal with multiple failures even with some of our contingency plans. Be rest assured that we didn't take this lightly, we have already devised a new game plan and will work hard towards making the registration process as fast and seamless as possible.
+
+We also know that most of you didn't get too excited with the talks we had... We hear you loud and clear and we'll make sure to have an even more strict vetting process for next year's speakers. We will leverage our, now even bigger, network of communities throughout Greece, speakers from the two previous DEVits and pick up every stone we can turn so we can book and verify the best and most challenging speakers for DEVit 2017.
+
+Other than those 2 major fails we have made over 50 bullet points on things that we should be improving, tweaking and fixing. You see, DEVit is a community organized event, everything that you see about DEVit comes out of our free time. However this community is not comprised by hobbyists, we are professionals with decades of experience in project management, web development, design, marketing and all the arts, crafts and sciences that can make this work. Combine that with our excess amounts of OCD and perfectionism and you can be rest assured that you are up for a big treat for 2017.
+
+## Looking ahead, 2017 here we come!
+
+One of the problems of DEVit 2016 was that we started planning too late, we only really got started in January 2016, barely 5 months before the event. That amount of time was ridiculously small considering everything that happens for DEVit happens at our free time... That will no longer be the case...
+
+We have decided on the dates of DEVit for the next three years and have started planning for all of them from now, right the next day when DEVit 2016 finished.
```

Comment:
right the next day when DEVit 2016 finished. -> as soon as DEVit 2016 finished. (if you don;t like this use: right after the day DEVit 2016 finished)